src/components/map/static-map.tsx

```diff
@@ -60,16 +82,48 @@ export function StaticMap() {
             );
           })}
         </div>
+        <div className="mx-auto mb-5 flex w-[90%] flex-row items-center justify-between">
+          <h2 className="mt-auto font-semibold">Mapa</h2>
+          <Dialog>
+            <DialogTrigger asChild className="hidden xl:flex">
```

Comment:
To nieptrzebnie jesli jest as child
```suggestion
            <DialogTrigger asChild>
```